app/test_split_integer.py

```diff
@@ -1,21 +1,104 @@
+import pytest
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value, number_of_parts, result",
+    [
+        (12, 1, 12),
+        (12, 2, 12),
+        (-12, 5, -12),
+        (12345, 2, 12345),
+        (12345, 6, 12345),
+        (12345, 100, 12345)
```

Comment:
The expected results for these test cases are incorrect. According to the task description, the `split_integer` function should return an array where the difference between the max and min number is <= 1, and the array should be sorted. Please revise the expected results to match these criteria.